### PR TITLE
Standardize on Bazel documentation link format

### PR DESCRIPTION
**Pull request by lrgoldstein**

### CODE REVIEW

File: analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/BuildWithoutTheBytesSuggestionProvider.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public SuggestionOutput getSuggestions(DataManager dataManager) {
         String recommendation =
             "Consider setting the Bazel flag --remote_download_minimal to minimize the data"
                 + " downloaded while using remote execution.\nAlso see"
-                + " https://docs.bazel.build/versions/main/command-line-reference.html"
+                + " https://bazel.build/reference/command-line-reference.html"
```
Comment (saraadams) [inline analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/BuildWithoutTheBytesSuggestionProvider.java:47]:
```suggestion
                + " https://bazel.build/reference/command-line-reference"
```

File: analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/CriticalPathNotDominantSuggestionProvider.java [hunk 2]
```diff
@@ -166,7 +166,7 @@ public SuggestionOutput getSuggestions(DataManager dataManager) {
                     + "An optimal speedup is expected by increasing the value to %d or"
                     + " more.\n"
                     + "Also see"
-                    + " https://docs.bazel.build/versions/main/command-line-reference.html"
+                    + " https://bazel.build/reference/command-line-reference.html"
```
Comment (saraadams) [inline analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/CriticalPathNotDominantSuggestionProvider.java:169]:
```suggestion
                    + " https://bazel.build/reference/command-line-reference"
```

File: analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/JobsSuggestionProvider.java [hunk 3]
```diff
@@ -83,7 +83,7 @@ public SuggestionOutput getSuggestions(DataManager dataManager) {
             "For local builds, setting the Bazel flag --jobs to a number is not"
                 + " recommended. Instead, omit this flag or set the value to \"auto\".\n"
                 + "Also see"
-                + " https://docs.bazel.build/versions/main/command-line-reference.html"
+                + " https://bazel.build/reference/command-line-reference.html"
```
Comment (saraadams) [inline analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/JobsSuggestionProvider.java:86]:
```suggestion
                + " https://bazel.build/reference/command-line-reference"
```

**Review by saraadams — APPROVED** [review]
Please also make sure to sign your commit. Also, you may want to check for other `.html` we can remove.

**Comment by lrgoldstein** [comment]
> Please also make sure to sign your commit. Also, you may want to check for other `.html` we can remove.

Done and done!

**Pull request merged**